R/lint.R

```diff
@@ -649,6 +649,72 @@ sarif_output <- function(lints, filename = "lintr_results.sarif") {
   write(jsonlite::toJSON(sarif, pretty = TRUE, auto_unbox = TRUE), filename)
 }
 
+#' Gitlab Report for lint results
```

Comment:
nit: the official name is `GitLab`, not `Gitlab`:

https://en.wikipedia.org/wiki/GitLab